Dockerfile

```diff
@@ -7,9 +7,12 @@ RUN pip install --user -r requirements.txt
 
 #App run ->Stage2
 FROM python:alpine
-COPY --from=baseimage /root/.local /root/.local
-COPY student.py /app/
 WORKDIR /app
+COPY --from=baseimage /root/.local /root/.local
+COPY student.py runapp.sh /app/
+COPY migrations /app/migrations/
```

Comment:
This could have been a single COPY statement

Reply:
I've combined multiple COPY commands in single command.